node/lib/util/submodule_rebase_util.js

```diff
@@ -75,6 +75,29 @@ const cleanupRebaseDir = co.wrap(function *(repo) {
     }
 });
 
+/**
+ * Make a new commit on the head of the specified `repo` having the same
+ * committer and message as the specified original `commit`, and return its
+ * sha.
+ *
+ * TODO: independent test
+ *
+ * @param {NodeGit.Repository} repo
+ * @param {NodeGit.Commit}     commit
+ * @return {String}
+ */
+exports.makeCommit = co.wrap(function *(repo, commit) {
+    assert.instanceOf(repo, NodeGit.Repository);
+    assert.instanceOf(commit, NodeGit.Commit);
+
+    const defaultSig = repo.defaultSignature();
+    const metaCommit = yield repo.createCommitOnHead([],
+                                                     defaultSig,
```

Comment:
I'm confused here.  Normally if Jones cherry-pick's Smith's commit, Smith is the author and Jones is the committer.

Reply:
No, I think you're right. I'll fix it.